src/ert/gui/simulation/run_dialog.py

```diff
@@ -96,9 +96,11 @@ def __init__(self, snapshot_model: SnapshotModel, parent: QWidget | None) -> Non
             # Only last section should be stretch
             horizontal_header.setSectionResizeMode(
                 section,
-                QHeaderView.Stretch
-                if section == horizontal_header.count() - 1
-                else QHeaderView.Interactive,
+                (
+                    QHeaderView.Stretch
+                    if section == horizontal_header.count() - 1
+                    else QHeaderView.Interactive
+                ),
```

Comment:
Just cosmetic?

Reply:
Yep, no behavior change

Reply:
Suggest reverting to the original then, as it is not related to the changes in the commit